components/mvp_table/commons/mvp_table.lua

```diff
@@ -0,0 +1,234 @@
+		for _, id in pairs(args.matchGroupIds) do
+			matchGroupIDConditions:add{ConditionNode(ColumnName('match2bracketid'), Comparator.eq, id)}
+		end
+	end
+
+	local tournamentConditions
+	if Table.isNotEmpty(args.tournaments) then
+		tournamentConditions = ConditionTree(BooleanOperator.any)
+		for _, tournament in pairs(args.tournaments) do
+			local page = mw.title.new(tournament)
+			tournamentConditions:add{
+				ConditionTree(BooleanOperator.all):add{
+					ConditionNode(ColumnName('pagename'), Comparator.eq, mw.ustring.gsub(page.text, ' ', '_')),
+					ConditionNode(ColumnName('namespace'), Comparator.eq, page.namespace),
+				},
+			}
+		end
+	end
+
+	local conditions = ConditionTree(BooleanOperator.all):add{
+		tournamentConditions,
+		matchGroupIDConditions,
+		--ConditionNode(ColumnName('extradata_mvp'), Comparator.neq, ''),
+	}
+
+	return conditions:toString()
+end
+
+function MvpTable._header1(args)
+	if String.isEmpty(args.title) then
+		return nil
+	end
+
+	local colspan = 2 + (args.points and 1 or 0)
+
+	return mw.html.create('tr')
+		:tag('th'):wikitext(args.title):attr('colspan', colspan):done():done()
+end
+
+function MvpTable._header2(args)
+	local header = mw.html.create('tr')
+		:tag('th'):wikitext('Player'):done()
+		:tag('th'):wikitext('#MVPs'):done()
+
+	if args.points then
+		header:tag('th'):wikitext('Points'):done()
+	end
+
+	return header:done()
+end
+
+function MvpTable._row(item, args)
+	local playerCell = mw.html.create('td')
+		:css('text-align', 'left')
+		:wikitext(Flags.Icon{flag = item.flag})
+		:wikitext('&nbsp;')
+		:wikitext('[[' .. item.name .. '|' .. item.displayName .. ']]')
+		:wikitext('&nbsp;')
+		:wikitext(Team.part(mw.getCurrentFrame(), item.team or ''))
+
+	local row = mw.html.create('tr')
+		:node(playerCell)
+		:tag('td'):wikitext(item.mvp):done()
+
+	if args.points then
+		row = mw.html.create('tr')
+			:tag('td'):wikitext(item.points):done()
+	end
+
+	return row:done()
+end
+
+-- overwritable function via /Custom
+function MvpTable.processData(queryData)
+	local playerList = {}
+	local mvpList = {}
+
+	for _, item in pairs(queryData) do
+		local mvp = (item.extradata or {}).mvp
+		if mvp then
+			for _, player in pairs(mvp.players or {}) do
+				if not playerList[player] then
+					playerList[player] = MvpTable.createPlayer(item.match2opponents, player)
+				end
+				playerList[player].mvp = playerList[player].mvp + 1
+				playerList[player].points = playerList[player].points + (mvp.points or 0)
+			end
+		end
+	end
+
+	for _, item in Table.iter.spairs(playerList, MvpTable.sortFunction) do
+		table.insert(mvpList, item)
+	end
+
+	return mvpList
+end
+
+-- overwritable function via /Custom
+function MvpTable.createPlayer(opponents, mvp)
+	local player = {
+		points = 0,
+		mvp = 0,
+		displayName = mvp,
+		name = mvp,
+	}
+
+	for _, opponent in pairs(opponents) do
+		local players = opponent.match2players or {}
+		mvp = MvpTable.pageFromMvp(mvp)
```

Comment:
![Screenshot 2022-06-03 10 46 22](https://user-images.githubusercontent.com/75081997/171821172-f884587d-6b8c-4c14-a732-c9cc8a64b5c0.png)
from: https://liquipedia.net/leagueoflegends/Special:LiquipediaDB/Mid-Season_Invitational/2022/Group_Stage

if you only set the link in the mvp param:
![Screenshot 2022-06-03 10 49 00](https://user-images.githubusercontent.com/75081997/171821778-fc1c021b-5063-4394-bac7-48f171b69427.png)

if you only set the displayname it will link to the wrong page

Reply:
Hm, I'd prefer if we in that case stored into LPDB differently? Table of dn, link and flag? Maybe add team here directly as well incase of a transfer?

Reply:
then this PR has to wait until the according adjustments are done in match2 for those wikis

Reply:
would mean we have to add a /custom for lots of wikis that we then have to kick later again
maybe lock it behind a switch param?

Reply:
Looks like it's already required? Considering that League has it's own custom?

Reply:
#1862 adjusts how mvp stuff is stored on lol
this module can now handle both versions
long term i think storing mvp the way i suggest for lol is preferable, accordingly i think switching all wikis with mvp stuff over to this might be a good idea (if PR 1862 is liked and merged that is)